### PR TITLE
Add github action for relase.

### PR DESCRIPTION
**Pull request by andreimejov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div align="center">
     <img src="https://github.com/frappe/design/blob/master/logos/logo-2019/frappe-charts-logo.png" height="128">
     <a href="https://frappe.github.io/charts">
-        <h2>Frappe Charts</h2>
+        <h2>Frappe Charts clone with fixes for Kotify Team</h2>
```
Comment (kalekseev) [inline README.md:4]: leave only this string and link to the original repo, nothing more

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```
Comment (kalekseev) [inline .github/workflows/release.yml]: configure your editor to always have a new line at the end of the file.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: Node.js Package
+
+on:
+  release:
+    types: [created]
```
Comment (kalekseev) [inline .github/workflows/release.yml:5]: what workflow you assume here?

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - run: npm run-script build
```
Comment (kalekseev) [inline .github/workflows/release.yml:15]: what the point of that step if the build results already stored in repo?

Comment (kalekseev) [reply .github/workflows/release.yml:15]: ok, I see, you removed build, shouldn't we run some test, does original lib have tests?

Comment (andreimejov) [reply .github/workflows/release.yml:15]: "test": "echo \"Error: no test specified\" && exit 1"

File: .github/workflows/release.yml (hunk 3 above)
Comment (andreimejov) [reply .github/workflows/release.yml:5]: publish on release creation, like in old edumate_gateway

**Review by kalekseev — APPROVED** [review]
Would be good to add npmignore file to not pack all repo files into the package.

**Pull request merged**